packages/@aws-cdk/aws-stepfunctions/test/distributed-map.test.ts

```diff
@@ -0,0 +1,606 @@
+import * as s3 from '@aws-cdk/aws-s3';
+import { CfnBucket } from '@aws-cdk/aws-s3';
+import * as cdk from '@aws-cdk/core';
+import * as stepfunctions from '../lib';
+import { CsvHeaders } from '../lib';
+
+describe('Distributed Map State', () => {
```

Comment:
Please add unit tests and modify / create an integration to verify that the deprecated `parameters` still work.

Reply:
I tried to do this and I couldn't figure out how to unit test a deprecation error.  I created a unit test for the parameters field and I expected it to still work, but I would get a deprecation warning that behaved as an error and failed the test.  I tried researching this behavior but my limited knowledge of deprecations and unit testing with typescript got the best of me.  Can you point me in the direction of any exampled for testing deprecations like this?

Reply:
I figured out how to add a test for the parameters field using the `testDeprecated` method.  I added this test to the map.test.ts module rather than to the distributed-map.test.ts module.  I figured that since Distributed Map is new, hopefully users will start of using the `itemSelector` field instead of `parameters`, but on the Map state the `parameters` field will be more prevalent.